.stickler.yml

```diff
@@ -0,0 +1,17 @@
+# add the linters you want stickler to use for this project
+linters:
+  eslint:
+    # indicate where is the config file for stylelint
+    config: './eslint.config'
```

Comment:
- This line of code right here is what's telling stickler what to listen to in order to get the ruleset in which it's gonna based its linting rules, so it is best if you set it up to be the same as the one you design in your eslint configuration so that the linting rules are the same in your development enviroment and the same in your PR checks.